CHANGELOG.md

```diff
@@ -0,0 +1,68 @@
+# Change Log
```

Comment:
I love this idea of doing this. Would it be appropriate to require contributors to add to the monthly change log? 

I say yes.